src/App.tsx

```diff
@@ -1,10 +1,72 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+interface Errors {
+  messages: string[];
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>();
+  const [errors, setErrors] = useState<Errors>({ messages: [] });
+  const [input, setInput] = useState('');
+  const [isLoadingTodos, setIsLoadingTodos] = useState(true);
+  const [isNotificationVisible, setNotificationVisible] = useState(false);
```

Comment:
This state is not really needed, you can display notification if there is any error in state on line 14